data/publiccloud/terraform/azure.tf

```diff
@@ -154,12 +154,12 @@ resource "azurerm_virtual_machine" "openqa-vm" {
         disable_password_authentication = true
         ssh_keys {
             path     = "/home/azureuser/.ssh/authorized_keys"
-            key_data = "${file("/root/.ssh/id_rsa.pub")}"
+            key_data = file("/root/.ssh/id_rsa.pub")
         }
     }
 
     tags = {
-        openqa_created_by = "${var.name}"
+        openqa_created_by = var.name
         openqa_created_date = "${timestamp()}"
         openqa_created_id = "${element(random_id.service.*.hex, count.index)}"
```

Comment:
just curious why this didn't need to be changed to 
`openqa_created_id = element(random_id.service.*.hex, count.index)`

Reply:
I went looking at the logs and fixing one by one and I might have missed it, but the new logs don't show a warning. In some cases, I removed the `"${` and got error... maybe this requires a string with quotes (I don't know)... Let's see in the next runs if we get warnings due to this line. Anyway, it won't cause any bug.

Reply:
agree!